cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -340,4 +340,78 @@ describe('User index view', () => {
       });
     });
   });
+
+  describe('User index view with the member_index_view feature flag enabled', () => {
```

Comment:
Note: While redundant, I've pulled this set of tests out into their own block, with their own setup for the time being since they require that the feature flag be enabled in order to run properly. That said, once this feature is no longer behind a feature flag, the actual tests within this block can be pulled out of here and moved into the "User actions" blocks within the overarching test.